### PR TITLE
Automated database population

### PR DESCRIPTION
**Pull request by gforge**
Thanks for a great test-example! 

As I wanted to quickly explore how sequelize behaves I added automated population of the database using the `faker` package, the user and the blogs have now 10 entries automatically populated. Considering that this is an "example" application it makes to have some test data available from the start. 

I also did some minor changes to the initialization structure for convenience for easier access to the database models.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -46,8 +42,34 @@ app.get('/', (req, res) => {
...
 
-// set port, listen for requests
-const PORT = process.env.PORT || 8080;
-app.listen(PORT, () => {
-  console.log(`Server is running on port ${PORT}.`);
-});
+intializedSequelize
+  // Clean tables
+  .sync({ force: true })
+  .then(async () => {
```
Comment (StefanNedelchev) [inline src/app.ts]: can we extract the fake DB population into a separate file and keep app.ts as clean as possible?

**Review by StefanNedelchev — CHANGES_REQUESTED** [review]

File: src/app.ts (hunk 1 above)
Comment (gforge) [reply src/app.ts]: Sure, I've moved the population into a separate file and also added a `npm run start:populate_db` - perhaps it should be mentioned in the README?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (StefanNedelchev) [inline .vscode/settings.json]: Thanks for doing the requested changes but please remove the settings.json file. Let's stick to the main purpose of the project and the PR. Enforcing specific theme colors for VSCode is not ok for me and it should be up to the users or the theme they have selected.

**Review by StefanNedelchev — CHANGES_REQUESTED** [review]
remove `.vscode/settings.json`

File: src/app.ts (hunk 1 above)
Comment (StefanNedelchev) [reply src/app.ts]: yes, you can add this to the README

File: .vscode/settings.json (hunk 2 above)
Comment (gforge) [reply .vscode/settings.json]: Sorry, this got committed without me thinking about it. I've dropped it and updated the README.

**Review by StefanNedelchev — APPROVED** [review]
Thanks for the PR and for doing the change requests. I think it's ready to be merged.

**Pull request merged**